### PR TITLE
Bugfix/json parse error

### PR DESCRIPTION
**Pull request by chungyau97**
Test
1. 	[done] AutoGPT
2. 	[done] BabyAGI
3. 	[done] Conversational Agent
4. 	[done] MRKL Agent LLM
5. 	[done] MRKL Agent Chat
6. 	[done] OpenAI Function Agent
7. 	[done] OpenaAPI Chain
8. 	[done] Conversational Retrieval QA Chain
9. 	[done] Conversation Chain
10. [done] LLM Chain
11. [done] Multi Prompt Chain
12. [done] Multi Retrieval QA Chain
13. [done] Retrieval QA Chain
14. [done] VectorDB QA Chain

### CODE REVIEW

File: packages/components/nodes/prompts/PromptTemplate/PromptTemplate.ts [hunk 1]
```diff
@@ -45,7 +45,20 @@ class PromptTemplate_Prompts implements INode {
...
         if (promptValuesStr) {
-            promptValues = JSON.parse(promptValuesStr.replace(/\s/g, ''))
+            promptValues = JSON.parse(promptValuesStr)
+            // example['something'] = {
+            //     "userId": 1,
+            //     "id": 1,
+            //     "title": example.title,
+            //     "done": true
+            // }
+            // example['arraySomething'] = [{
```
Comment (HenryHengZJ) [inline packages/components/nodes/prompts/PromptTemplate/PromptTemplate.ts:55]: commented code?

File: packages/components/nodes/agents/ConversationalAgent/ConversationalAgent.ts [hunk 2]
```diff
@@ -107,7 +107,8 @@ class ConversationalAgent_Agents implements INode {
             memory.chatHistory = new ChatMessageHistory(chatHistory)
             executor.memory = memory
         }
-        const result = await executor.call({ input })
+        const reverseInput = handleEscapeCharacters(input, true)
```
Comment (HenryHengZJ) [inline packages/components/nodes/agents/ConversationalAgent/ConversationalAgent.ts:110]: I think `reverseInput` might be a lil bit confusing. Maybe `escapedCharInput`?

File: packages/components/nodes/prompts/PromptTemplate/PromptTemplate.ts [hunk 3]
```diff
@@ -45,7 +45,20 @@ class PromptTemplate_Prompts implements INode {
 
         let promptValues: ICommonObject = {}
         if (promptValuesStr) {
-            promptValues = JSON.parse(promptValuesStr.replace(/\s/g, ''))
+            promptValues = JSON.parse(promptValuesStr)
```
Comment (HenryHengZJ) [inline packages/components/nodes/prompts/PromptTemplate/PromptTemplate.ts:48]: need to apply changes to `ChatPromptTemplate` and `FewShotPromptTemplate` and test

**Comment by Justinandjohnson** [comment]
I'm still having an issue with this.. help

**Comment by Justinandjohnson** [comment]
I'm still getting:

ErrorWithoutStackTrace: PineconeClient: Error getting project name: SyntaxError: Unexpected token A in JSON at position 0

**Comment by chungyau97** [comment]
> I'm still getting:
> 
> ErrorWithoutStackTrace: PineconeClient: Error getting project name: SyntaxError: Unexpected token A in JSON at position 0

After you swap into `bugfix/JsonParseError` branch, you still get error? 
Could you share your flow and input message?

**Comment by Justinandjohnson** [comment]
Hmm maybe the issue is i don't know how to swap into that branch. Apolgizes.

**Comment by Justinandjohnson** [comment]
Screen Shot 2023-07-23 at 1 30 00 PM
238d19c0-39ac-472f-8572-843d7ab0fe35">
I'm on the latest branch but still getting this error.

**Comment by Justinandjohnson** [comment]
Screen Shot 2023-07-23 at 1 31 49 PM

**Comment by raffareis** [comment]
>  I'm on the latest branch but still getting this error.

@Justinandjohnson This is not merged into main yet. Idk how you are running flowise, but if it is locally, and from a forked FlowiseAI/Flowise, you could try this command on the repo folder in your machine:

```bash
git fetch upstream
git checkout -b bugfix/JsonParseError upstream/bugfix/JsonParseError
```

And if you didn't fork, but rather is using FlowiseAI/Flowise directly:
```bash
git fetch origin
git checkout bugfix/JsonParseError
```

If everything went right, you will be on the bugfix/JsonParseError branch that should fix this problem.

**Comment by HenryHengZJ** [comment]
we have discarded the changes here, and merged this here instead - https://github.com/FlowiseAI/Flowise/pull/612

**Pull request closed without merging**